price_matching_system/user.py

```diff
@@ -0,0 +1,91 @@
+from flask import Blueprint
+from flask import flash
+from flask import g
+from flask import redirect
+from flask import render_template
+from flask import request
+from flask import session
+from flask import url_for
+from datetime import date, datetime, timedelta
+from werkzeug.security import check_password_hash
+from werkzeug.security import generate_password_hash
+import pyodbc
+
+from price_matching_system.db import get_conn
+
+bp = Blueprint("user", __name__)
+
+@bp.route("/user")
+def user():
+    """After user login.
+    Allow user to change the person information.
+    Submit button allows user to save and back to user page.
+    """
+
+    conn = get_conn()
+    cursor = conn.cursor()
+    query = f"""SELECT * FROM dbo.tbl_User u
+                    WHERE UserId = {session.get('login_id')}"""
```

Comment:
login id is not the same as user id
use user id from the g object here

Reply:
userId is from the table and the POST data should be connected to the table.

Reply:
fixed